CMakeLists.txt

```diff
@@ -46,7 +46,7 @@
 ###########################################################################
 
 # Visual Studio 2017 requires 3.7.
```

Comment:
nit: s/requires 3.7/requires 3.7 or greater/ ?

Reply:
I just deleted this line: we don't use 2017 anymore (on 2019).